src/Core/Framework/App/Command/ActivateAppCommand.php

```diff
@@ -9,7 +9,7 @@
 use Symfony\Component\Console\Attribute\AsCommand;
 
 /**
- * @internal only for use by the app-system, will be considered internal from v6.4.0 onward
+ * @final only for use by the app-system, will be considered internal from v6.4.0 onward
```

Comment:
Commands should also all be internal, we don't want to keep backwards compatibility on the code level, but we want to keep the backward compatibility on the CLI level, as that's how you should execute the commands and not per php